content/exporters/custom-exporter/Java/Metrics.md

```diff
@@ -0,0 +1,312 @@
+---
+title: "Metrics exporter"
+draft: false
+weight: 3
+aliases: [/custom_exporter/java/stats, /guides/custom_exporter/java/stats]
+---
+
+- [Introduction](#introduction)
+- [Implementation](#implementation)
+- [Runnable example](#runnable-example)
+- [References](#references)
+
+## Introduction
+
+A metrics exporter must extend the abstract class [MetricExporter](https://github.com/census-instrumentation/opencensus-java/blob/master/exporters/metrics/util/src/main/java/io/opencensus/exporter/metrics/util/MetricExporter.java) implementing the `export` method:
+which for purposes of brevity is:
+
+```java
+import java.util.Collection;
+
+import io.opencensus.exporter.metrics.util.MetricExporter;
+
+public void export(Collection<Metric> metrics);
+```
+
+The sole method `export` will be used to process and translate a collection of [Metric](https://static.javadoc.io/io.opencensus/opencensus-api/0.19.2/io/opencensus/metrics/export/Metric.html) to your desired monitoring backend's data.
+
+After an exporter is created, it must be wrapped in a [MetricProducer](https://static.javadoc.io/io.opencensus/opencensus-api/0.19.2/io/opencensus/metrics/export/MetricProducer.html) class, which is registered with [MetricProducerManager.add](https://static.javadoc.io/io.opencensus/opencensus-api/0.19.2/io/opencensus/metrics/export/MetricProducerManager.html#add-io.opencensus.metrics.export.MetricProducer-). In the example constructor
```

Comment:
Consider using www.javadoc.io instead of static.javadoc.io.

Reply:
IMO you don't need to mention `MetricProducer` here. Exporter is more about consuming the metrics and don't need to worry about the producers. If no producers are registered, you can still implement and create exporters though they won't see metrics coming in.

Reply:
Removed mention of MetricProducer in new push.

For the JavaDoc links, www is OK for the top level JavaDoc but for individual classes I get 404s with s/static/www/ For example,
https://www.javadoc.io/io.opencensus/opencensus-exporter-metrics-util/0.19.2/io/opencensus/exporter/metrics/util/MetricExporter.html

is not found. So, I did not change the links.